spec/mailers/reply_forwarding_mailer_spec.rb

```diff
@@ -57,7 +57,7 @@
           filename: 'a_file.txt',
           byte_size: 298,
         )
-        inbound_email.create_raw_email_attachment!(blob:, content_type: 'text/plain')
+        inbound_email.raw_email_attachment.update!(blob:, content_type: 'text/plain')
```

Comment:
It took a long time to figure out that this change gets the spec passing again. Not sure exactly what's going on here. Some sort of autosave callback issue.